ts/parts/Tooltip.ts

```diff
@@ -1386,8 +1413,8 @@ H.Tooltip.prototype = {
                 label.height,
                 point
             ),
-            anchorX = (point.plotX as any) + chart.plotLeft,
-            anchorY = (point.plotY as any) + chart.plotTop,
+            anchorX = point.plotX! + chart.plotLeft,
```

Comment:
It is generally not a good idea. Usually if you have to make assertions it shows a flaw in your code. The only reason for using them is in some heavy async stuff, but not here. If it gets not solved with type checks, you should cast to any, so that we can work on it later on. We should consider a lint rule to prevent assertions.

Reply:
Solved by 7efea0d